spec/features/user_index_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+describe 'Filter users by key', type: :feature do
+  let(:users) { FactoryBot.create_list(:user, 3) }
+  let(:user) { FactoryBot.create :user }
+
+  before do
+    users
+    sign_in user
+    visit users_path
+  end
+
+  it 'is possible to add all users except one' do
```

Comment:
The description does not match the test case

Reply:
it does since we have all users and the `user` so seeing all users would be users.length+1 but fair it is not a good naming